### PR TITLE
Persist time queries when clicking on breadcrumbs links

### LINKED ISSUE #718: Dates are not persisted when using the breadcrumbs
**To Reproduce**

1. Go to `/wp-admin/admin.php?page=wc-admin#/analytics/orders`.
2. Change the time range.
3. Click on _Analytics_ in the breadcrumb.

Expected: time range is persisted.

Actual: time range is lost.

**Screenshots**
imatge

### PR DESCRIPTION
**Pull request by Aljullu**
Fixes #718.

### Screenshots
_(this PR shouldn't have produced any visible change)_
imatge

### Detailed test instructions:
- Go to `/wp-admin/admin.php?page=wc-admin#/analytics/products` and change the time period.
- Click on _Analytics_ in the breadcrumbs of the top bar.
- Verify the time period is still the same.

### CODE REVIEW

File: client/lib/nav-utils/index.js [hunk 1]
```diff
@@ -6,6 +6,11 @@ import history from 'lib/history';
 import { parse, stringify } from 'qs';
 import { isEmpty, pick, uniq } from 'lodash';
 
+/**
+ * Internal dependencies
+ */
+import * as navUtils from './index';
```
Comment (psealock) [inline client/lib/nav-utils/index.js]: Below we are using `getQuery` directly. I didn't know you could do this. I think its easier to see if used directly (relies on hoisting).

Comment (psealock) [reply client/lib/nav-utils/index.js]: Oh is this for `jest.mock` to work. Maybe add a comment?

**Review by psealock — APPROVED** [review]
Testing well!

File: client/lib/nav-utils/index.js (hunk 1 above)
Comment (Aljullu) [reply client/lib/nav-utils/index.js]: Yes, exactly, this seems to be the only way to mock functions from the same module.

Added a comment in 6dd3c7f9c452cd29f7815020bb3be2bdb2f23d7f.

**Pull request merged**